From 4f9b7a86873ca65476fa8b711f08a8db46aa7a95 Mon Sep 17 00:00:00 2001
From: Peter Chen <peter.chen@freescale.com>
Date: Thu, 4 Jun 2015 12:31:34 +0800
Subject: [PATCH] MLK-10994-2 ARM: imx6ul-14x14-ddr3-arm: remove dts entry for
 adc

ADC needs to rework at ARM board, so it needs another dts
to avoid conflict with USBOTG1.

Signed-off-by: Peter Chen <peter.chen@freescale.com>
---
 arch/arm/boot/dts/imx6ul-14x14-ddr3-arm2.dts | 7 -------
 1 file changed, 7 deletions(-)

diff --git a/arch/arm/boot/dts/imx6ul-14x14-ddr3-arm2.dts b/arch/arm/boot/dts/imx6ul-14x14-ddr3-arm2.dts
index 4e484da..acb4336 100644
--- a/arch/arm/boot/dts/imx6ul-14x14-ddr3-arm2.dts
+++ b/arch/arm/boot/dts/imx6ul-14x14-ddr3-arm2.dts
@@ -70,13 +70,6 @@
 	};
 };
 
-&adc1 {
-	pinctrl-names = "default";
-	pinctrl-0 = <&pinctrl_adc1>;
-	vref-supply = <&reg_vref_3v3>;
-	status = "okay";
-};
-
 &ecspi1 {
 	fsl,spi-num-chipselects = <1>;
 	cs-gpios = <&gpio4 26 0>;
-- 
1.8.0

